content/en/docs/contribute/style/style-guide.md

```diff
@@ -44,22 +44,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 ### Use upper camel case for API objects
 
-When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
+When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal case. You may see different capitalization, such as "configMap", in the  [API Reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/). When writing general documentation, it's better to use upper camel case, calling it "ConfigMap" instead.
+
+When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
+
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. 
 
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-Refer to API objects without saying "object," unless omitting "object"
-leads to an awkward construction.
+The following examples focus on capitalization. Review the related guidance on [Code Style](#code-style-inline-code) for more information on formatting API objects. 
 
-{{< table caption = "Do and Don't - API objects" >}}
+{{< table caption = "Do and Don't - Use Pascal case for API objects" >}}
 Do | Don't
 :--| :-----
-The pod has two containers. | The Pod has two containers.
-The HorizontalPodAutoscaler is responsible for ... | The HorizontalPodAutoscaler object is responsible for ...
 A PodList is a list of pods. | A Pod List is a list of pods.
-The two ContainerPorts ... | The two ContainerPort objects ...
-The two ContainerStateTerminated objects ... | The two ContainerStateTerminateds ...
+The `Volume` object contains a `hostPath` field. | The volume object contains a hostPath field.
+Every `ConfigMap` is part of a namespace. | Every configMap is part of a namespace. 
+For managing confidential data, consider using the Secret API. | For managing confidential data, consider using a secret.
```

Comment:
:wave: @geoffcline .

Does the `Secret` example represent this capitalization case:
`When you are generally discussing an API object, use [sentence-style capitalization]`

Personal opinion: I am not convinced the site needs to format API resources and definitions as code.
To be consistent, any example in this guide should format API objects with the recommended style.
Otherwise, if there are remaining questions about formatting, raise an issue. Lines 62-64 add new content and should follow the style guide. Lines 62 and 63 display an API resource, definition, and field name in code format.

Would this example fit with the current style and capitalization recommendations?
```
For managing confidential data, consider using a `Secret`. | For managing confidential data, consider using a secret.
```

Reply:
Thank you for reviewing again. 

Did you see my discussion on this issue with Qiming? https://github.com/kubernetes/website/pull/24645#discussion_r540642351

I don't think our style guide *mandates* code style yet, so I wanted to illustrate the different options.